ui/src/components/uploader/QasUploader.vue

```diff
@@ -286,7 +297,15 @@ export default {
       }
 
       const clonedValue = extend(true, [], this.modelValue)
-      const numberIndex = this.modelValue.findIndex(file => this.getFileName(file) === index)
+      const numberIndex = this.modelValue.findIndex(file => {
+        if (this.useObjectModel) {
+          return file.uuid
+            ? file.uuid === index
+            : file.url.includes(index)
```

Comment:
- Cabe na mesma linha: `return file.uuid ? .uuid === index : file.url.includes(index)`
- Por que essa lógica?

Reply:
Corrigido. Preciso verificar se ele possui um uuid e se ele tiver e for igual ao valor que vem no index(Posso mudar esse nome, o index é o id do S3) quer dizer que eu tenho arquivos no uploader que foram populados pelo back. Agora se eu faço um uploader de um arquivo e deleto ele, ele não vai ter um uuid pq ainda não foi para o back, nesse caso preciso pegar o index( id que o S3 cria pra ele) e verificar se ele contem na url. Antes das alterações que fiz o index não voltava o index de fato e sim o id que o S3 cria, por exemplo `159784655.pdf` , posso alterar para outro nome?